src/mpi/request/request_impl.c

```diff
@@ -5,12 +5,7 @@
 
 #include "mpiimpl.h"
 
-#if !defined(MPIR_REQUEST_PTR_ARRAY_SIZE)
-/* use a larger default size of 64 in order to enhance SQMR performance */
-#define MPIR_REQUEST_PTR_ARRAY_SIZE 64
-#endif
-
-int MPIR_Cancel(MPIR_Request * request_ptr)
+int MPIR_Cancel_impl(MPIR_Request * request_ptr)
 {
     int mpi_errno = MPI_SUCCESS;
```

Comment:
Can you separate the request related refactoring into a separate commit, likely before the autogeneration commit?

Reply:
done